bindings/rust/wasmedge-sys/src/lib.rs

```diff
@@ -168,20 +170,34 @@ pub use vm::Vm;
 
 use wasmedge_types::{error, WasmEdgeResult};
 
-/// Type alias for a boxed native function.
+/// Type alias for a boxed native function. This type is used in thread-safe cases.
 pub type BoxedFn = Box<dyn Fn(Vec<WasmValue>) -> Result<Vec<WasmValue>, u8> + Send + Sync>;
 
 lazy_static! {
     static ref HOST_FUNCS: Arc<Mutex<HashMap<usize, BoxedFn>>> =
         Arc::new(Mutex::new(HashMap::with_capacity(
-            std::env::var("MAX_HOST_FUNC_LENGTH")
+            env::var("MAX_HOST_FUNC_LENGTH")
                 .map(|s| s
                     .parse::<usize>()
-                    .expect("MAX_HOST_FUNC_LENGTH should be a positive integer."))
+                    .expect("MAX_HOST_THREADED_FUNC_LENGTH should be a positive integer."))
```

Comment:
The variable name is not the same as L179. Could you check which name should be used here?
If the name of environment variable is changed, should it be in the changelog?

Reply:
Fixed by using `MAX_HOST_FUNC_LENGTH`.